### PR TITLE
Add illustration of connect via hostname, password, etc.

### PR DESCRIPTION
**Pull request by EnergyBoyTW**
When I am using this library, I cannot find the demonstration of how can I connect with hostname and password before I read the source code.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -86,6 +86,8 @@ Simple consumer:
 
 
     async def main(loop):
+        # Connect with the givien parameters is also valiable.
+        # aio_pika.connect_robust(host="host", login="login", password="password")
```
Comment (mosquito) [inline README.rst:90]: I think it's important to mention that this is mutually exclusive.

Comment (EnergyBoyTW) [reply README.rst:90]: @mosquito  What's that mean ? sorry I am new in contributor

Comment (mosquito) [reply README.rst:90]: You can pass `url` or `kw`-based params not both.

Comment (EnergyBoyTW) [reply README.rst:90]: Yes of course

**Review by mosquito — APPROVED** [review]

**Comment by EnergyBoyTW** [comment]
PR completed.

**Comment by EnergyBoyTW** [comment]
@mosquito should I wait for you to merge into master branch before I close the PR ?

**Review by Alviner — APPROVED** [review]

**Review by mosquito — APPROVED** [review]

**Pull request merged**